Brokerages/Zerodha/ZerodhaBrokerage.cs

```diff
@@ -263,7 +248,12 @@ private bool Unsubscribe(IEnumerable<Symbol> symbols)
         /// <returns> Quote</returns>
         public Quote GetQuote(Symbol symbol)
         {
-            var instrument = _symbolMapper.GetZerodhaInstrumentToken(symbol.ID.Symbol, symbol.ID.Market);
+            var instrumentTokenList = _symbolMapper.GetZerodhaInstrumentTokenList(symbol.ID.Symbol);
```

Comment:
`GetZerodhaInstrumentTokenList` says `Converts an Zerodha symbol to a List of Zerodha Instrument Tokens available from various exchange` but we are actually sending in the Lean symbol?

Reply:
Right, sorry! Error in the comments